tasks/kernel_matrix_testing/infra.py

```diff
@@ -130,3 +151,37 @@ def ssh_key_to_path(ssh_key):
         ssh_key_path = find_ssh_key(ssh_key)
 
     return ssh_key_path
+
+
+def ask_for_ssh():
+    return (
+        ask(
+            "You may want to provide ssh key, since the given config launches a remote instance.\nContinue witough ssh key?[Y/n]"
```

Comment:
```suggestion
            "You may want to provide a ssh key, since the given config launches a remote instance.\nContinue without a ssh key?[Y/n]"
```